### ISSUE TITLE
Character Limit Lengths

### ISSUE THREAD

**Issue by MichaelDalrymple** [wet-boew/wet-boew-styleguide#333]
A note that on http://wet-boew.github.io/wet-boew-styleguide/v4/writing/strctr-en.html#image, it states that alt text should be under 125 characters. However the Canada.ca style guide states 140. I suspect the Style Guide takes precedence in these matters, so this may need to be updated.

**Comment by duboisp** [comment]
For publishing GoC web content the [Canada.ca style guide](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html) should have precedence.
